cmd/config_test.go

```diff
@@ -52,9 +62,42 @@ func TestPasswordConfig(t *testing.T) {
 func TestTLSConfigLoad(t *testing.T) {
 	null := "/dev/null"
 	nonExistent := "[nonexistent]"
-	cert := "../test/grpc-creds/creds-test/cert.pem"
-	key := "../test/grpc-creds/creds-test/key.pem"
-	caCert := "../test/grpc-creds/minica.pem"
+	cert := path.Join(os.TempDir(), "TestTLSConfigLoad.cert.pem")
+	key := path.Join(os.TempDir(), "TestTLSConfigLoad.key.pem")
+	caCert := path.Join(os.TempDir(), "TestTLSConfigLoad.cacert.pem")
```

Comment:
```suggestion
	cert := path.Join(t.TempDir(), "TestTLSConfigLoad.cert.pem")
	key := path.Join(t.TempDir(), "TestTLSConfigLoad.key.pem")
	caCert := path.Join(t.TempDir(), "TestTLSConfigLoad.cacert.pem")
```

`t.TempDir()` will cleanup after itself, unlike `os.TempDir()` which leaves orphaned files in the container according to
```
...
	cmd := exec.Command("ls", "-al", "/tmp")
	stdout, _ := cmd.Output()
	fmt.Println(string(stdout))
...

$ ./t.sh -u -p ./cmd -f TestTLSConfigLoad -v -w
drwxrwxrwt 1 root root 4096 May 14 14:26 ./
drwxr-xr-x 1 root root 4096 May 14 14:26 ../
---------- 1 root root  461 May 14 14:26 TestTLSConfigLoad.cacert.pem
---------- 1 root root  473 May 14 14:26 TestTLSConfigLoad.cert.pem
---------- 1 root root  207 May 14 14:26 TestTLSConfigLoad.key.pem
```